ibm_catalog.json

```diff
@@ -285,9 +304,91 @@
 							"default_value": "__NULL__",
 							"description": "The CRN of an existing KMS instance to use in this solution. If not set, a new KP instance is provisioned.",
 							"required": false
+						},
+						{
+							"key": "existing_en_instance_crn",
```

Comment:
put the full name of the service in variable - event_notification

Reply:
we will address these changes in dev repo (to not increase merge conflicts) with new PR. Issue created https://github.ibm.com/GoldenEye/issues/issues/10462